### PR TITLE
Ch add permission banner rc

### PR DESCRIPTION
**Pull request by acedeywin**
Primary reviewer:

## Description

New feature: This PR adds a new permission for the remove case banner

### Checklist
- [ ] Corresponding issue has been opened
- [x] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Related Issues
Fixes #....

### CODE REVIEW

**Review by stephenhand — COMMENTED** [review]
Do we not also need the checks for the 'remove' link for the 'Added to Case' banners?

**Comment by acedeywin** [comment]
> Do we not also need the checks for the 'remove' link for the 'Added to Case' banners?

@stephenhand - It's done.

File: plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx [hunk 1]
```diff
@@ -85,13 +97,20 @@ const ContactAddedToCaseBanner: React.FC<Props> = ({
       <Text>
         <Template code="CaseMerging-ContactAddedTo" />
       </Text>
-      <CaseLink type="button" onClick={() => viewCaseDetails(connectedCase)} data-fs-id="LinkedCase-Button">
+      <CaseLink
```
Comment (stephenhand) [inline plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx:100]: We still want to allow users to view the case if they don't have permission to remove it - we need to check PermissionActions.VIEW_CASE to determine if the link is active

**Review by stephenhand — COMMENTED** [review]
Still 1 more issue

File: plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx (hunk 1 above)
Comment (acedeywin) [reply plugin-hrm-form/src/components/caseMergingBanners/ContactAddedToCaseBanner.tsx:101]: @stephenhand - It's done.

**Review by stephenhand — APPROVED** [review]

**Pull request merged**